pkg/cloudconfig/cloudconfig.go

```diff
@@ -0,0 +1,132 @@
+package cloudconfig
```

Comment:
This one is quite simple, I've tried one from Juju project and it was too complicated and had messy deps.